controlplane/kubeadm/config/crd/bases/controlplane.cluster.x-k8s.io_kubeadmcontrolplanes.yaml

```
@@ -28,7 +37,7 @@ spec:
     status: {}
   validation:
     openAPIV3Schema:
-      description: KubeadmControlPlane is the Schema for the KubeadmControlPlane API.
+      description: KubeadmControlPlane is the Schema for the KubeadmControlPlane API.Age
```

```suggestion
      description: KubeadmControlPlane is the Schema for the KubeadmControlPlane API.
```
Don't think this change is needed.

---

Not sure how it got appended here.